PluginsAndFeatures/azure-toolkit-for-intellij/rider/test-integration/org/cases/functions/coreTools/FunctionsCoreToolsManagerTestCore.kt

```diff
@@ -0,0 +1,162 @@
+/**
+ * Copyright (c) 2022 JetBrains s.r.o.
+ *
+ * All rights reserved.
+ *
+ * MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+ * documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+ * to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all copies or substantial portions of
+ * the Software.
+ *
+ * THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO
+ * THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package org.cases.functions.coreTools
+
+import com.intellij.ide.util.PropertiesComponent
+import com.intellij.openapi.util.io.FileUtil
+import com.jetbrains.rdclient.util.idea.toIOFile
+import com.jetbrains.rider.test.asserts.shouldBe
+import com.jetbrains.rider.test.asserts.shouldBeNull
+import com.jetbrains.rider.test.asserts.shouldContains
+import com.jetbrains.rider.test.asserts.shouldNotBeNull
+import com.jetbrains.rider.test.base.BaseTestWithSolution
+import com.microsoft.intellij.configuration.AzureRiderSettings
+import org.jetbrains.plugins.azure.functions.coreTools.FunctionsCoreToolsConstants
+import org.jetbrains.plugins.azure.functions.coreTools.FunctionsCoreToolsManager
+import org.testng.annotations.*
+import java.io.File
+
+abstract class FunctionsCoreToolsManagerTestCore(
```

Comment:
Let's follow Rider's base test class naming convention here: `<some_logicl_name>TestBase.kt`.

Reply:
Done, updated other tests in one go.